client/hotrod-client/src/test/java/org/infinispan/client/hotrod/query/pressure/LargePutAllPressureTest.java

```diff
@@ -38,13 +49,17 @@ protected EmbeddedCacheManager createCacheManager() throws Exception {
       return TestCacheManagerFactory.createServerModeCacheManager(config);
    }
 
+   protected boolean useTransactions() {
+      return false;
+   }
+
    @Override
    protected org.infinispan.client.hotrod.configuration.ConfigurationBuilder
       createHotRodClientConfigurationBuilder(String host, int serverPort) {
       org.infinispan.client.hotrod.configuration.ConfigurationBuilder builder =
             super.createHotRodClientConfigurationBuilder(host, serverPort);
 
-      builder.socketTimeout(SIZE * 2);
+      builder.socketTimeout(TIMEOUT);
```

Comment:
We can avoid coming back to the timeout. If you add:

```
   @Override
   protected HotRodServer createHotRodServer() {
      return HotRodTestingUtil.startHotRodServer(cacheManager, "127.0.0.1", ServerTestingUtil.findFreePort(),
            new HotRodServerConfigurationBuilder(), false);
   }
```

To override the HotRod server creation to disable the test handler, this should finish waaaay more quickly. The test handler is designed to check the replay of requests a single byte at a time.

Reply:
Thanks @jabolina, maybe we can apply it here: https://github.com/infinispan/infinispan/pull/12905

Reply:
I still think we need to rethink the timeout value for bulk operations like getAll, putAll, iterator and add client listener with include current state....

Reply:
Great. Makes more sense to me.

Reply:
I created https://issues.redhat.com/browse/ISPN-16658 to track that